tamr_unify_client/models/project/estimated_pair_counts.py

```diff
@@ -0,0 +1,42 @@
+from tamr_unify_client.models.base_resource import BaseResource
+
+
+class EstimatedPairCounts(BaseResource):
+    """Estimated Pair Counts info for Mastering Project"""
+
+    @classmethod
+    def from_json(cls, client, resource_json, api_path=None) -> "EstimatedPairCounts":
+        return super().from_data(client, resource_json, api_path)
+
+    @property
+    def is_up_to_date(self) -> bool:
+        """Whether an estimate pairs job has been run since the last edit to the binning model.
+
+        :type: bool
+        """
+        return self._data.get("isUpToDate")
+
+    @property
+    def total_estimate(self) -> dict:
+        """The total number of estimated candidate pairs and generated pairs for the model across all clauses.
+
+        :type: dict[str, int]
```

Comment:
Left comments about these type declaration in #132 before realizing they should have been on this PR. But those changes should be made in this PR.